script.js

```diff
@@ -161,3 +161,72 @@ injectMarkup();
 modalContainer.addEventListener("click", (e) => {
   closeModal(e);
 });
+
+// SUBMIT FORM
+const form = document.querySelector(".form");
+const feedbackEl = document.querySelector(".form-feedback");
+const SUBMIT_LINK = "https://formspree.io/f/myyvwvbw";
+
+const clearUpNotice = () => {
+  setTimeout(() => {
+    feedbackEl.classList.remove("email-success");
+    feedbackEl.classList.remove("email-fail");
+    feedbackEl.textContent = "";
+  }, 3000);
+};
+
+const showSuccess = () => {
+  feedbackEl.classList.remove("email-fail");
+  feedbackEl.classList.add("email-success");
+  clearUpNotice();
+};
+
+const showError = () => {
+  feedbackEl.classList.remove("email-success");
+  feedbackEl.classList.add("email-fail");
+  clearUpNotice();
+};
+
+const handleSubmit = async (event) => {
+  event.preventDefault();
+
+  // const name = event.target.name.value;
+  const email = event.target.email.value;
+  // const message = event.target.message.value;
+
+  const emailRegex = /^[a-z]+@[a-z0-9-]+\.[a-z0-9-.]+$/;
+
+  if (!emailRegex.test(email)) {
+    showError();
+    feedbackEl.textContent = "Invalid email";
+    return;
+  }
```

Comment:
- [OPTIONAL] It would be better if the user can be given a comprehensive message that would direct him or her to what the error is and what to correct in order for the error not to keeping occurring when the user has no knowledge of what is expected of him or her.